lihzahrd/header/anglerquestfish.py

```diff
@@ -43,7 +43,11 @@ class AnglerQuestFish(enum.IntEnum):
     TROPICAL_BARRACUDA = 38
 
     # none of the other tools know what fish this is ... *shrug*
+    # we now know it's a SCARAB_FISH; UNKNOWN1 remains for back-compatibility.
     UNKNOWN1 = 39
+    SCARAB_FISH = 39
```

Comment:
I'm swapping these two around, so that `SCARAB_FISH` gets displayed instead of `UNKNOWN1` for `AnglerQuestFish(39)`!